Branch-SDK/Branch-SDK/BNCContentDiscoveryManager.m

```diff
@@ -71,61 +71,255 @@ - (NSString *)standardSpotlightIdentifierFromActivity:(NSUserActivity *)userActi
+                  description:(NSString *)description
+            publiclyIndexable:(BOOL)publiclyIndexable
+                         type:(NSString *)type
+                 thumbnailUrl:(NSURL *)thumbnailUrl
+                     keywords:(NSSet *)keywords
+                     userInfo:(NSDictionary *)userInfo
+               expirationDate:(NSDate*)expirationDate
+                     callback:(callbackWithUrl)callback {
+    [self indexContentWithTitle:title
+                    description:description
+              publiclyIndexable:publiclyIndexable
+                           type:type
+                   thumbnailUrl:thumbnailUrl
+                       keywords:keywords
+                       userInfo:userInfo
+                 expirationDate:nil
+                       callback:callback
+              spotlightCallback:(callbackWithUrlAndSpotlightIdentifier)callback];
 }
 
-- (void)indexContentWithTitle:(NSString *)title description:(NSString *)description publiclyIndexable:(BOOL)publiclyIndexable type:(NSString *)type thumbnailUrl:(NSURL *)thumbnailUrl keywords:(NSSet *)keywords userInfo:(NSDictionary *)userInfo expirationDate:(NSDate *)expirationDate callback:(callbackWithUrl)callback {
+
+- (void)indexContentWithTitle:(NSString *)title
+                  description:(NSString *)description
+            publiclyIndexable:(BOOL)publiclyIndexable
+                         type:(NSString *)type
+                 thumbnailUrl:(NSURL *)thumbnailUrl
+                     keywords:(NSSet *)keywords
+                     userInfo:(NSDictionary *)userInfo
+                     callback:(callbackWithUrl)callback {
+    [self indexContentWithTitle:title
+                    description:description
+              publiclyIndexable:publiclyIndexable
+                           type:type
+                   thumbnailUrl:thumbnailUrl
+                       keywords:keywords
+                       userInfo:userInfo
+                 expirationDate:nil
+                       callback:callback
+              spotlightCallback:nil];
+}
+
+- (void)indexContentWithTitle:(NSString *)title
+                  description:(NSString *)description
+            publiclyIndexable:(BOOL)publiclyIndexable
+                         type:(NSString *)type
+                 thumbnailUrl:(NSURL *)thumbnailUrl
+                     keywords:(NSSet *)keywords
+                     userInfo:(NSDictionary *)userInfo
+            spotlightCallback:(callbackWithUrlAndSpotlightIdentifier)spotlightCallback {
+    [self indexContentWithTitle:title
+                    description:description
+              publiclyIndexable:publiclyIndexable
+                           type:type
+                   thumbnailUrl:thumbnailUrl
+                       keywords:keywords
+                       userInfo:userInfo
+                 expirationDate:nil
+                       callback:nil
+              spotlightCallback:(callbackWithUrlAndSpotlightIdentifier)spotlightCallback];
```

Comment:
shouldn't need to cast here, aka should be able to remove `(callbackWithUrlAndSpotlightIdentifier)`